hrms/hr/doctype/interview/interview.js

```diff
@@ -54,9 +54,22 @@ frappe.ui.form.on("Interview", {
 		);
 
 		if (allow_feedback_submission) {
-			frm.page.set_primary_action(__("Submit Feedback"), () => {
-				frm.trigger("submit_feedback");
-			});
+			frappe.db
```

Comment:
Interview Feedback can be submitted by multiple interviewers. This just checks for a single feedback. So even if 1 interviewer has submitted the feedback, the next one won't see a button at all. Should also check if interviewer is the session user.